docs/install_guide.adoc

```diff
@@ -0,0 +1,206 @@
+| ccadeae4322346fe9feee6bb0677b014 | regionOne | nova         | compute      | True    | internal  | http://nova-internal.openstack.svc:8774/v2.1             |
+| dfac9cb0706d411b8c90c6df264fb575 | regionOne | nova         | compute      | True    | public    | https://nova-public-openstack.apps-crc.testing/v2.1      |
+| e8314601b5184f74b99b0815e1d57d5e | regionOne | cinderv3     | volumev3     | True    | internal  | http://cinder-internal.openstack.svc:8776/v3             |
++----------------------------------+-----------+--------------+--------------+---------+-----------+----------------------------------------------------------+
+----
+
+Now verify that Telemetry operator with Prometheus metric storage is ready:
+
+[,console]
+----
+$ oc get telemetry
+NAME        STATUS   MESSAGE
+telemetry   True     Setup complete
+
+$ oc get metricstorage
+NAME             STATUS   MESSAGE
+metric-storage   True     Setup complete
+
+$ oc get route metric-storage-prometheus
+NAME                        HOST/PORT                                              PATH   SERVICES                    PORT   TERMINATION     WILDCARD
+metric-storage-prometheus   metric-storage-prometheus-openstack.apps-crc.testing          metric-storage-prometheus   web    edge/Redirect   None
+----
+
+Prometheus metrics will be viewable in a web browser at the `HOST/PORT` address returned above. ie:
+https://metric-storage-prometheus-openstack.apps-crc.testing.
+
+== Installing the Operator
+
+.Procedure
+
+. Now that you have a ready working environment, you can install the Watcher Operator.
+First, create a `watcher-operator.yaml` file:
++
+[source,yaml]
+----
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: watcher-operator-index
+  namespace: openstack-operators
+spec:
+  image: quay.io/openstack-k8s-operators/watcher-operator-index:latest
+  sourceType: grpc
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openstack
+  namespace: openstack-operators
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: watcher-operator
+  namespace: openstack-operators
+spec:
+  name: watcher-operator
+  channel: alpha
+  source: watcher-operator-index
+  sourceNamespace: openstack-operators
+----
++
+. With cluster-admin permissions in Openshift run:
++
+[,console]
+----
+$ oc apply -f watcher-operator.yaml
+catalogsource.operators.coreos.com/watcher-operator-index created
+operatorgroup.operators.coreos.com/openstack unchanged
+subscription.operators.coreos.com/watcher-operator created
+----
++
+. Check that the operator is installed:
++
+[,console]
+----
+$ oc get subscription.operators.coreos.com/watcher-operator -n openstack-operators
+NAME               PACKAGE            SOURCE                   CHANNEL
+watcher-operator   watcher-operator   watcher-operator-index   alpha
+
+$ oc get pod -l openstack.org/operator-name=watcher -n openstack-operators
```

Comment:
If you modify the PR, it would be technically better to check the status (I know I gave this one before :) ) with:

$ oc get csv watcher-operator.v0.0.1

Example output:

`NAME                      DISPLAY            VERSION   REPLACES   PHASE
watcher-operator.v0.0.1   Watcher Operator   0.0.1                Succeeded`